src/models/actions/Http.js

```diff
@@ -45,7 +45,7 @@ class Http extends BaseAction {
     try {
       transport = context.get("__http_transport");
     } catch (e) {
-      if (global.settings.ignoreSSL === "false") {
+      if (global.settings?.ignoreSSL === "false" || !global.settings) {
```

Comment:
This line here broke the RG in the webhook context.

In that specific context, the RapidTest.executeTest method is imported ther the test-service and used locally outside the worker itself. In that context there are not global settings so this object does not exist. 

this code was changed several months ago, but an update back to the testing-service was not done until some time later. Once that update was made, this broke.

An api test to the /proxy endpoint will alert us if this breaks again and we should follow up this PR with such a test.